Source/ViewControllers/BrickViewController+UIViewControllerPreviewing.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  BrickViewController+ UIViewControllerPreviewing.swift
+//  BrickKit
+//
+//  Created by Aaron Sky on 8/2/17.
+//  Copyright © 2017 Wayfair. All rights reserved.
+//
+
+import UIKit
+
+#if os(iOS)
+extension BrickViewController: UIViewControllerPreviewingDelegate {
+    open func previewingContext(_ previewingContext: UIViewControllerPreviewing, viewControllerForLocation location: CGPoint) -> UIViewController? {
+        guard let indexPath = brickCollectionView.indexPathForItem(at: location) else {
```

Comment:
This is not unit-tested. Any chance that we could create a unit test, where we pass in the `center` property of a brick? So if we have a brick e.g. `GenericBrick<Label>(height: .fixed(size: 100))`, we know that the point `CGPoint(x: 160, y: 50)` (given a width of 320) should return the correct view controller, with the `sourceBrick` set